autoload/vista/finder/fzf.vim

```diff
@@ -86,11 +92,11 @@ function! s:Run(...) abort
 endfunction
 
 function! s:Highlight() abort
-  syntax match FZFVistaBracket /\[\|\]/ contained
   syntax match FZFVistaNumber /^\s*\zs\d*:\ze\w/
-  syntax match FZFVistaTag    /^\s.*\ze\[/ contains=FZFVistaNumber
-  syntax match FZFVistaScope  /^\s.*\[.*\]\ze\s/ contains=FZFVistaTag,FZFVistaBracket
-  syntax match FZFVista /^.*$/ contains=FZFVistaBracket,FZFVistaNumber,FZFVistaTag,FZFVistaScope
+  syntax match FZFVistaTag    /^[^\[]*\(\[\)\@=/ contains=FZFVistaNumber
+  syntax match FZFVistaScope  /^[^]]*]/ contains=FZFVistaTag,FZFVistaBracket
+  syntax match FZFVista /^[^│┌]*[│┌└\n]/ contains=FZFVistaBracket,FZFVistaNumber,FZFVistaTag,FZFVistaScope
```

Comment:
To be honest, I never really master these opaque patterns :(. I don't fully understand `/^[^│┌]*[│┌└\n]/`, but it does work great.

Reply:
Looking at that with fresh eyes it should be `^[^│┌└]*` (fixed)

The highlighter still is not optimal (has theoretical bugs), I'll send a different pull request when I get a chance.